_data/compatibility/casa.yaml

```diff
@@ -0,0 +1,61 @@
+---
+name: Casa
+internal_url: /en/compatibility/casa
+logo: /img/compatibility/casa/casa.png
+rbf:
+  tested:
+    date: "2019-10-24"
+    platforms:
+      - iOS
+      - Android
+    version: "2.9.0"
+  features:
+    receive:
+      notification: "false"
+      list: "false"
+      details: "false"
+      shows_replaced_version: "false"
+      shows_original_version: "false"
+    send:
+      signals_bip125: "false"
+      list: "false"
+      details: "false"
+      shows_replaced_version: "false"
+      shows_original_version: "false"
+  examples:
+    - image: /img/compatibility/casa/rbf/approve-send-transaction.png
+      caption: >
+        Sending Transaction - Default send transaction screen. No RBF options.
+    - image: /img/compatibility/casa/rbf/transaction-list.png
+      caption: >
+        Receiving Replacement Transaction - Since no unconfirmed transaction
+        appears, neither original nor replacement show until the replacement
+        transactions confirms.
+segwit:
+  tested:
+    date: "2019-10-24"
+    platforms:
+      - iOS
```

Comment:
Similar to above.

Reply:
ah, OK... I'd go with iOS since it's the one I used for screenshots

Reply:
added a commit for this change